install.yaml

```diff
@@ -0,0 +1,27 @@
+name: ddev-drupal
+ddev_version_constraint: ">= v1.23.0"
+
+# Files copied into the project root preserving paths
+# The dist folder structure mirrors the target project structure
+project_files:
+  - config.wunderio.yaml
+  - docker-compose.wunderio.yaml
+# - drush/sites/local.site.yml
+
+global_files:
```

Comment:
@zHelmet I discovered this install.yml trick - it moves the `commands` and `wunderio/core/` under your host ~/.ddev/ folder to be global. `wunderio/core/`  I had to manually mount in docker-composer.wunderio.yaml.  So instead of using .gitignore to hide the files, they'll be always there. I think this sounds promissing?

<img width="1046" height="907" alt="Screenshot 2025-11-11 at 08 45 01" src="https://github.com/user-attachments/assets/bd3f9859-3ffe-4123-a49f-8b8f3b2ee789" />

 I don't have yet full picture as I don't have this as package yet. I'd like to try and create that, then try out versioning and maybe if possible add some scripts that would check version, update the packege, make sure old files are removed.